### ISSUE TITLE
[Shipping methods] Minimal amount to apply a discount is incorrectly handled

### ISSUE THREAD

**Issue by audez** [openfoodfoundation/wishlist#323]

**Comment by audez** [comment]
oh if it's on purpose there should be a big warning/explanation.. because as such it's terribly misleading

**Comment by tschumilas** [comment]
At present,  I'm pretty sure ANY OFN fees -  whether they are in the OC incoming,  OC outgoing,  or whether they are a shipping fee or a payment fee....      are calculated on the product total.   No fees are calculated on fees.     I agree it would be good to have some kind of a warning would be good.    I think incepting this will be interesting because some users want fees on fees,  and others don't,   and maybe it depends on the type of fee...     Agreeing though that it is particularly confusing for price sac.

**Comment by lin-d-hop** [comment]
@audez The (English) user guide has a warning about this. Discounts as a feature has never been 'delivered'. We've just found a hack that works and we all use it.
This issue will be addressed within the Vouchers and Credits work.

**Comment by audez** [comment]
yep @lin-d-hop thanks, i'll add the same warning in the FR guide too. Cool if it is addressed later! 

Meanwhile i think it could really help to reword the translations, something like: 
- "Minimal amount" ---> "Minimum basket amount (total products cost without enterprise fees)"
- "Discount amount" --> "Discount fee" 
- "Normal amount"  ---> "Normal fee"
 
@RachL would you agree with the following translations on the FR side ?
![Screen Shot 2022-11-18 at 14 09 08](https://user-images.githubusercontent.com/28386648/202712791-88f89a63-b81a-453a-a550-4582f2052d25.png)

![Screen Shot 2022-11-18 at 14 11 49](https://user-images.githubusercontent.com/28386648/202713088-7cb19053-a4d8-40dd-b6d0-100039ab75af.png)

It would also be nice to have an "info" icon next to the fields, to give further explanation.
